data/js/values/content4.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+    var answer = document.querySelector('#answer span');
+    var button = document.querySelector('button');
+
+    addon.port.on('tokens', function(tokens) {
+        for (var i = 0, l = tokens.length; i < l; i++) {
+            // set the token container to active and shows the token
+            document.querySelector('#' + tokens[i]).classList.add('active');
+        }
+    });
+
+    button.addEventListener('click', function() {
+        // highlight the answer
+        answer.classList.add('answer');
```

Comment:
It seems like we'd probably want to change the selected radio button to the correct one, and deselect any others at this point as well.